fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java

```diff
@@ -287,8 +287,8 @@ private String genTypes() {
 
     private String genHeader(Map<String, String> properties) {
         String header = "";
-        if (properties.containsKey("format")) {
-            String headerType = properties.get("format");
+        if (properties.containsKey("csv_format")) {
```

Comment:
do not change this key, should compatible with 2.0. 
We should check its value, if it is not valid then throw exception.

Reply:
done